Source/WebCore/platform/mediastream/libwebrtc/LibWebRTCProvider.h

```diff
@@ -106,10 +107,8 @@ class WEBCORE_EXPORT LibWebRTCProvider : public WebRTCProvider {
     WEBCORE_EXPORT void disableEnumeratingAllNetworkInterfaces();
     WEBCORE_EXPORT void enableEnumeratingAllNetworkInterfaces();
     bool isEnumeratingAllNetworkInterfacesEnabled() const;
-
-    bool m_enableEnumeratingAllNetworkInterfaces { false };
-    // FIXME: Remove m_useNetworkThreadWithSocketServer member variable and make it a global.
-    bool m_useNetworkThreadWithSocketServer { true };
+    WEBCORE_EXPORT void enableEnumeratingVisibleNetworkInterfaces();
+    bool isEnumeratingVisibleNetworkInterfacesEnabled() const { return m_enableEnumeratingVisibleNetworkInterfaces; }
```

Comment:
Style checker asks that you move this into the implementation, since this entire class is `WEBCORE_EXPORT`ed.